pkg/segment/utils/segconsts.go

```diff
@@ -148,6 +148,11 @@ var VERSION_TSGFILE = []byte{0x01}
 var VERSION_MBLOCKSUMMARY = []byte{0x01}
 
 var VERSION_SEGSTATS = []byte{2}
+var VERSION_SEGSTATS_LEGACY = []byte{1}
+
```

Comment:
I partially agree with this. It might be good to mention what `VERSION_SEGSTATS` vs `VERSION_SEGSTATS_BUF` means, but I don't think it's necessary to comment on `VERSION_SEGSTATS` vs `VERSION_SEGSTATS_LEGACY`